tests/components/teslemetry/__init__.py

```diff
@@ -9,19 +10,39 @@
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers import entity_registry as er
 
-from .const import CONFIG
-
 from tests.common import MockConfigEntry
 
 
+def mock_config_entry(expires_at: int | None = None) -> MockConfigEntry:
+    """Create a mock config entry."""
+    if expires_at is None:
+        expires_at = int(time.time()) + 3600
+
+    return MockConfigEntry(
+        domain=DOMAIN,
+        version=2,
+        unique_id="abc-123",
+        data={
+            "token": {
+                "access_token": "test_access_token",
+                "refresh_token": "test_refresh_token",
+                "expires_at": expires_at,
+            },
+        },
+    )
+
+
 async def setup_platform(
-    hass: HomeAssistant, platforms: list[Platform] | None = None
+    hass: HomeAssistant,
+    platforms: list[Platform] | None = None,
+    expires_at: int | None = None,
 ) -> MockConfigEntry:
     """Set up the Teslemetry platform."""
 
-    mock_entry = MockConfigEntry(
-        domain=DOMAIN, data=CONFIG, minor_version=2, unique_id="abc-123"
-    )
+    if expires_at is None:
+        expires_at = int(time.time()) + 3600
+
+    mock_entry = mock_config_entry(expires_at)
```

Comment:
you now have the same default behavior twice

Reply:
Yes I do, removed.